packages/moonstone/Divider/Divider.js

```diff
@@ -92,6 +94,13 @@ const DividerBase = kind({
 	render: (props) => {
 		delete props.spacing;
 
+		deprecate({
```

Comment:
This will cause the deprecation to warn on every render which is rather noisy. Would like to limit it to a single invocation is possible.

Maybe something like: https://codesandbox.io/s/weathered-fire-9xo6z

Reply:
updated